api/osf_api.py

```diff
@@ -133,6 +134,27 @@ def delete_project(session, guid, user=None):
             n.get()
             n.delete()
 
+
+def create_custom_collection(session):
```

Comment:
Nit-picking: this is not a test function but a utility method. Please add a simple DocStr for it. Same for the `delete_custom_collections()`.